app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::ItemsController < ApplicationController
+
+  def index
+    render json: ItemSerializer.new(Item.all)
+  end
+  
+  def show 
+    render json: ItemSerializer.new(Item.find(params[:id]))
+  end
+
+  def create 
+    id = Item.last.id ||= 0
+    params = item_params.merge(id: id+1)
+    render json: ItemSerializer.new(Item.create(params))
```

Comment:
Found that rails driver was trying to assign the item to an existing spot. Not sure if this is best practice